### PR TITLE
handle multiple DOIs

### PR DESCRIPTION
**Pull request by maxulysse**
Replace half of https://github.com/nf-core/sarek/pull/1494

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test tests/ --verbose --profile +docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
waiting on https://github.com/nf-core/modules/pull/5556 as well

**Review by jasmezz — DISMISSED** [review]
LGTM from what I understand from the string-replacing syntax. If you prefer a really close check of those lines in the main.nf, you should wait for another review. But comparing those to the nf-core subworkflow update and your changed code, it looks alright to me ;)

**Review by jasmezz — APPROVED** [review]

**Pull request merged**
